src/ResourceManagement/Network/Microsoft.Azure.Management.Network/Generated/VirtualNetworkGatewaysOperationsExtensions.cs

```diff
@@ -292,13 +292,13 @@ public static IPage<VirtualNetworkGateway> List(this IVirtualNetworkGatewaysOper
             /// <param name='virtualNetworkGatewayName'>
             /// The name of the virtual network gateway.
             /// </param>
-            /// <param name='parameters'>
-            /// Parameters supplied to the Begin Reset Virtual Network Gateway operation
-            /// through Network resource provider.
+            /// <param name='gatewayVip'>
+            /// Vip parameter supplied to the Begin Active-Active Reset Virtual Network
+            /// Gateway operation through Network resource provider.
             /// </param>
-            public static VirtualNetworkGateway Reset(this IVirtualNetworkGatewaysOperations operations, string resourceGroupName, string virtualNetworkGatewayName, VirtualNetworkGateway parameters)
+            public static VirtualNetworkGateway Reset(this IVirtualNetworkGatewaysOperations operations, string resourceGroupName, string virtualNetworkGatewayName, string gatewayVip = default(string))
```

Comment:
Parameter VirtualNetworkGateway parameters was not used so removed it and added new optional param:string gatewayVip